tests/test_return_class.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from src.return_class import *
+
+
+class Test_ReturnClass(unittest.TestCase):
+    def test_return_number(self):
+        example = ReturnClass()
+        actual_value = example.return_number()
+        self.assertEqual(actual_value, 897)
```

Comment:
We were talking about making this number here into variable like 
```suggestion
        expected_value = 897
        self.assertEqual(actual_value, expected_value)
```